_posts/2023-09-18-the-definitive-guide-2-cn.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title:  "Chinese edition of Trino: The Definitive Guide"
+author: Manfred Moser, Martin Traverso, Matt Fuller
+image: /assets/ttdg2-cn-cover.png
+excerpt_separator: <!--more-->
+show_pagenav: false
+---
+
+Trino, Trino, Trino everywhere. Just looking at our website stats and the users
+in our community chat, we know that Trino is going places. We also know that one
+of these places with a large user community is China. And now we have good news
+for you. A translatio of the second edition of the book to Chinese is now
+available.
+
+<!--more-->
+
+Today, we are happy to announce that a translation of the book [Trino: The
+Definitive Guide]({{ site.url }}/trino-the-definitive-guide.html) to Chinese is
+now available for the communities all across China and far beyond. We invite you
```

Comment:
```suggestion
now a lower barrier to read for native speakers. We invite you
```

Reply:
Added this instead of replacing the other part.